src/ngamsPlugIns/ngamsPlugIns/eso/ngamsExtractFitsHdrDppi.py

```diff
@@ -233,9 +234,9 @@ class (ngamsServer).
         for f in fils:
             cmd = constructCommand(f, head, struct, skey, tsv, xmlfl, mode, check)
             logger.debug('Executing command: %s', cmd)
-            stat, result, _ = execCmd(cmd)
+            stat, result, stderr = execCmd(cmd)
             if stat != 0:
-                errMsg = "Processing of header for file %s failed: %s" % (filename, result)
+                errMsg = "Processing of header for file %s failed: %s" % (filename, stderr)
```

Comment:
This should help seeing the actual problems (`printhead` prints errors to stdout). It requires adding `utils` to `from ngamsLib.ngamsCore import ....` at the top of the file.

```suggestion
                errMsg = "Processing of header for file %s failed:\nstdout:%s\nstderr:%s" % (filename, utils.b2s(result), utils.b2s(stderr))
```